dgc-testdata-tests/Program.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace dgc.testdata.tests
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            await DGCTestDataVerifier.TestAll();
```

Comment:
name - won't list seperately from now on, will assume you just do a ReSharper name refactor or equivalant